BLaurenB/techtime
# Admin can view any order through admin_order_path

#### What's this PR do?
Admin can now view any order because they are routed through the admin order path instead of the order path.

Previously, the link to view an order from the admin orders index was to the order_path.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
app/views/admin/orders/index.html.erb to see the path change

spec/features/orders/admin_can_view_orders_on_dashboard_spec.rb updated test to match the path

#### How should this be manually tested?
log in as admin and click on an order that's been placed. you should be able to view any order.

#### Any background context you want to provide?
#### What are the relevant tickets?
Closes #143 
#### Screenshots (if appropriate)
#### Questions:
